Casks/g/ganttproject.rb

```diff
@@ -1,24 +1,19 @@
 cask "ganttproject" do
-  version "3.2.3200,3.2.3230"
-  sha256 "4e6ebd308378d3d1936050ff6407630692d5b8833f05f139428b052cf91982dc"
+  arch arm: "silicon", intel: "intel"
 
-  url "https://github.com/bardsoftware/ganttproject/releases/download/ganttproject-#{version.csv.first}/ganttproject-#{version.csv.second}.dmg",
+  version "3.3.3300"
+  sha256 arm:   "4260ea2f2f7b7e960aab13729ce71b231a38059064d1e82a787867124994150c",
+         intel: "2119348b2d5357925d4d522e3d01e3af12d398654edd1c90734d5f9b34c75c67"
+
+  url "https://github.com/bardsoftware/ganttproject/releases/download/ganttproject-#{version}/ganttproject-#{version}-#{arch}.dmg",
       verified: "github.com/bardsoftware/ganttproject/"
   name "GanttProject"
   desc "Gantt chart and project management application"
   homepage "https://www.ganttproject.biz/"
 
   livecheck do
-    url :url
-    regex(%r{/ganttproject[._-]v?(\d+(?:\.\d+)+)/ganttproject[._-]v?(\d+(?:\.\d+)+)\.dmg$}i)
-    strategy :github_latest do |json, regex|
-      json["assets"]&.map do |asset|
-        match = asset["browser_download_url"]&.match(regex)
-        next if match.blank?
-
-        "#{match[1]},#{match[2]}"
-      end
-    end
+    url :stable
```

Comment:
`homebrew-cask` is slightly different default to `homebrew-core`

```suggestion
  url :url
```